applications/projects/SofaPhysicsAPI/src/SofaPhysicsAPI/SofaPhysicsSimulation.cpp

```diff
@@ -395,32 +395,17 @@ int SofaPhysicsSimulation::load(const char* cfilename)
     std::string filename = cfilename;
     sofa::helper::BackTrace::autodump();
 
-    //bool wasAnimated = isAnimated();
     sofa::helper::system::DataRepository.findFile(filename);
     m_RootNode = m_Simulation->load(filename.c_str());
     if (m_RootNode.get())
     {
         sceneFileName = filename;
         m_Simulation->init(m_RootNode.get());
         return updateOutputMeshes();
```

Comment:
In fact could you keep returning the result of outputMeshes to not break my version:

int result = API_SUCCESS;
if (m_RootNode.get())
{
...
   result = updateOutputMeshes();
} 
else
{
  m_RootNode = m_Simulation->createNewGraph("");
  return API_SCENE_FAILED;
}
...
return result;